### DISCUSSION TITLE
IP address and file request logging

### DISCUSSION THREAD

**Discussion by nicheath** [static-web-server/static-web-server#77] [Ideas]
I have been looking through the documentation and have been trying to find a way to log what IP requests what file. It doesn't seem possible yet but I wanted to make sure.

**Comment by joseluisq** [comment]
Yeah, It's not supported at the moment but as new feature why not.
So feel free to rise a feature request.

**Comment by joseluisq** [comment]
I have included support for logging file requests with their remote addresses. See #112
